OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+SizeSemanticTokens.swift

```diff
@@ -2,152 +2,105 @@
+    @objc open var sizeIconWithHeadingSmallSizeMediumMobile: SizeSemanticToken { DimensionRawTokens.dimension500 }
+    @objc open var sizeIconWithHeadingSmallSizeMediumTablet: SizeSemanticToken { DimensionRawTokens.dimension500 }
+    @objc open var sizeIconWithHeadingSmallSizeShortMobile: SizeSemanticToken { DimensionRawTokens.dimension400 }
+    @objc open var sizeIconWithHeadingSmallSizeShortTablet: SizeSemanticToken { DimensionRawTokens.dimension400 }
+    @objc open var sizeIconWithHeadingSmallSizeTallMobile: SizeSemanticToken { DimensionRawTokens.dimension550 }
+    @objc open var sizeIconWithHeadingSmallSizeTallTablet: SizeSemanticToken { DimensionRawTokens.dimension550 }
+    @objc open var sizeIconWithHeadingXlargeSizeMediumMobile: SizeSemanticToken { DimensionRawTokens.dimension600 }
+    @objc open var sizeIconWithHeadingXlargeSizeMediumTablet: SizeSemanticToken { DimensionRawTokens.dimension700 }
+    @objc open var sizeIconWithHeadingXlargeSizeShortMobile: SizeSemanticToken { DimensionRawTokens.dimension550 }
+    @objc open var sizeIconWithHeadingXlargeSizeShortTablet: SizeSemanticToken { DimensionRawTokens.dimension650 }
+    @objc open var sizeIconWithHeadingXlargeSizeTallMobile: SizeSemanticToken { DimensionRawTokens.dimension650 }
+    @objc open var sizeIconWithHeadingXlargeSizeTallTablet: SizeSemanticToken { DimensionRawTokens.dimension800 }
+    @objc open var sizeIconWithLabelLargeSizeMedium: SizeSemanticToken { DimensionSemanticTokens.dimension350 }
+    @objc open var sizeIconWithLabelLargeSizeShort: SizeSemanticToken { DimensionSemanticTokens.dimension300 }
+    @objc open var sizeIconWithLabelLargeSizeShorter: SizeSemanticToken { DimensionSemanticTokens.dimension250 }
+    @objc open var sizeIconWithLabelLargeSizeTall: SizeSemanticToken { DimensionSemanticTokens.dimension400 }
+    @objc open var sizeIconWithLabelLargeSizeTaller: SizeSemanticToken { DimensionSemanticTokens.dimension550 }
+    @objc open var sizeIconWithLabelMediumSizeMedium: SizeSemanticToken { DimensionSemanticTokens.dimension250 }
+    @objc open var sizeIconWithLabelMediumSizeShort: SizeSemanticToken { DimensionSemanticTokens.dimension200 }
+    @objc open var sizeIconWithLabelMediumSizeTall: SizeSemanticToken { DimensionSemanticTokens.dimension300 }
+    @objc open var sizeIconWithLabelSmallSizeMedium: SizeSemanticToken { DimensionSemanticTokens.dimension200 }
+    @objc open var sizeIconWithLabelSmallSizeShort: SizeSemanticToken { DimensionSemanticTokens.dimension150 }
+    @objc open var sizeIconWithLabelSmallSizeTall: SizeSemanticToken { DimensionSemanticTokens.dimension250 }
+    @objc open var sizeIconWithLabelXlargeSizeMedium: SizeSemanticToken { DimensionSemanticTokens.dimension500 }
+    @objc open var sizeIconWithLabelXlargeSizeShort: SizeSemanticToken { DimensionSemanticTokens.dimension400 }
+    @objc open var sizeIconWithLabelXlargeSizeTall: SizeSemanticToken { DimensionSemanticTokens.dimension550 }
```

Comment:
Be careful, in some keys, there's an extra "Size" that you didn't have before.

Reply:
@julien-deramond Your PR is not working ; you must replace any instance of `DimensionSemanticTokens` by `DimensionRawTokens`.
The code does not compile (see for exemple the checks on the commit like [the build check](https://github.com/Orange-OpenSource/ouds-ios/actions/runs/12029675364/job/33535458257))
![Capture d’écran 2024-11-26 à 13 34 24](https://github.com/user-attachments/assets/2d9225f8-4a5e-4745-a44a-3b300f879a98)